stable/appmesh-controller/values.yaml

```diff
@@ -21,6 +21,17 @@ sidecar:
     requests:
       cpu: 10m
       memory: 32Mi
+    # sidecar.resources/limits: Envoy CPU and memory limits
+    limits:
```

Comment:
Looks good, can you please also update the README configuration section [1] 

[1] https://github.com/aws/eks-charts/blob/master/stable/appmesh-controller/README.md